libcxx/include/__type_traits/is_trivially_relocatable.h

```diff
@@ -23,14 +22,8 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 // A type is trivially relocatable if a move construct + destroy of the original object is equivalent to
 // `memcpy(dst, src, sizeof(T))`.
-
-#if __has_builtin(__is_trivially_relocatable)
-template <class _Tp, class = void>
-struct __libcpp_is_trivially_relocatable : integral_constant<bool, __is_trivially_relocatable(_Tp)> {};
-#else
```

Comment:
I think we agree that we want to re-introduce this at some point? I think I'd just add a `&& 0` with a comment to this or some issue. Otherwise someone will probably come along and think "hey, there is a builtin for this in clang" and try to re-add this.